docs/Manage-Space.rst

```diff
@@ -293,28 +298,69 @@ One of the two following cases occurs when you select this tab.
 
 |image27|
 
+.. _SpaceHome:
+
+**Space home**
+
+The default space home page contains the following content:
+
+-  Activity stream: displays the space's activities
+-  Description: displays the space's description and the list of the space's managers
+-  :ref:`Calendar <CalendarApp>`: displays the events of the space's calendar by day (today's events by default)
+-  :ref:`Who is on line? <WhoIsOnlineApp>`: displays the online members of the space.
+
+.. note:: :ref:`Calendar <CalendarApp>` and :ref:`Who is on line? <WhoIsOnlineApp>` portlets are the same
+          as for :ref:`intranet homepage <PLFHomepage>` but specific for the space's calendar and members.
+
+Just after the space's creation, only two portlets appear in the newly created space:
```

Comment:
Shouldn't we merge the 2 parts, it seems still a bit redundant to me. We could insert the screenshot below each bullet in the list. And we can add that the Who is online? portlet appears onlilne when at least one member is online (which explains why it can be not visible). WDYT ?

Reply:
Ok